docs/source/manufacturing/3d_printing/troubleshooting/troubleshooting.rst

```diff
@@ -0,0 +1,9 @@
+Troubleshooting Common Issues
+=============================
+
+Here are some hyperlinks to common issues with printers that are worth looking at during a troubleshooting process.
+
+* `Underextrusion and Overextrusion <../specific_skill_guides/temperature_tuning/temperature_tuning.html>`_
```

Comment:
This link, and the rest of the links on this page, attempt to directly reference the html directory hierarchy. We try not to do this - direct html linking to internal pages, I mean (this referencing is a form of directly linking, assuming we know where Sphinx will put the files in the final html hierarchy). 

Instead, if you truly want to use direct references, use the :doc: command instead of direct linking (with the underscore) to reference internal 
RST docs using the repo hierarchy. 

As an example, I would rewrite this line as:
```
*  :doc:`Underextrusion and Overextrusion <../specific_skill_guides/temperature_tuning/temperature_tuning>`
```

The remaining links would be very similar.